sirepo/template/srw.py

```diff
@@ -1991,6 +1991,11 @@ def _generate_srw_main(data, plot_reports, beamline_info):
             ]
         )
     else:
+        if report == "multiElectronAnimation":
```

Comment:
It seems odd to not set v.wm in all cases. What is it in the third case?

Reply:
@moellep if `report == "multiElectronAnimation"` and  `run_all == True`  do we want to set `v.wm == False`?

Reply:
When you export the python source from the cogwheel, all the options should be explicit in main() - all sequential reports should be True, and ```v.wm = False```. I think ```run_all == True``` in that case. Each plot can export its own python source, and in that case, the only setting which should be on, is the specific plot - so ```v.wm = False``` should not be present in that case.

Try it out on each plot in the UI to verify.